src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/ResolvedJavaType.java

```diff
@@ -365,6 +366,13 @@ default ResolvedJavaMethod[] getDeclaredMethods(boolean forceLink) {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns a list containing all the non-static methods present within this type.
```

Comment:
Point out that the returned list is unmodifiable (like the API for `Stream.toList()` does).